### ISSUE TITLE
Persist index after unmounting file system

### ISSUE THREAD

**Issue by danielkrajnik** [cboxdoerfer/fsearch#366]
Index seems to remove entries of a google drive file system if it gets temporarily unmounted (google drive using rclone). Due to some security/technical issues google drive doesnt mount on boot and needs to be mounted manually. Fsearch seems to forget about files in this folder when that happens. Would that be possibly related and could an option to persist index for a certain amount of time be added?

**Comment by cboxdoerfer** [comment]
Yes, a feature which solves that is planned for the 0.3 release. This will add something like an "archive" option for folders, which means FSearch will keep the index of that folder, even when the folder vanishes from the system. That should be especially useful for external or remote drives as well.

**Comment by danielkrajnik** [comment]
That's brilliant news, looking forward to 0.3 then.

**Comment by cboxdoerfer** [comment]
@SvenAbels that's difficult to estimate. 0.2 is planned for August, then I'll start working on the features for 0.3. Since it's quite some work it probably won't be ready by the end of this year, so maybe spring or summer of 2023.

**Comment by edisondotme** [comment]
See #263
